index.html

```diff
@@ -0,0 +1,35 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <meta http-equiv="X-UA-Compatible" content="ie=edge">
+  <title>Unit 2 Assessment</title>
+  <script src="./index.js" type="text/javascript"></script>
+  <link rel="stylesheet" href="index.css">
+</head>
+<body>
+  <div class="header">
+  
+  <img class="img" src="/Users/bsihame/Desktop/core/unit2/unit_2_assessment/unit_2_assessment/logo.png">
```

Comment:
For this image you needed to use the link we shared with you in the README. It was a link on the first bullet point of the README. Click the link and note the url once the image opens. That was the url we needed here.